code/modules/antagonists/revenant/revenant.dm

```diff
@@ -49,7 +49,6 @@
 	move_resist = MOVE_FORCE_OVERPOWERING
 	mob_size = MOB_SIZE_TINY
 	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
-	flags_1 = RAD_NO_CONTAMINATE_1
```

Comment:
There are no more barriers to cross. All I have in common with the uncontrollable and the insane, the vicious and the evil, all the mayhem I have caused and my utter indifference toward it I have now surpassed. My pain is constant and sharp and I do not hope for a better world for anyone, in fact I want my pain to be inflicted on others. I want no one to escape, but even after admitting this there is no catharsis, my punishment continues to elude me and I gain no deeper knowledge of myself; no new knowledge can be extracted from my telling. This confession has meant nothing.